lib/events/playback.go

```diff
@@ -134,7 +137,69 @@ func WriteForPlayback(ctx context.Context, sid session.ID, reader AuditReader, d
 			log.WithError(err).Warningf("Failed to close writer.")
 		}
 	}()
-	return w.Write(ctx)
+	return w.chunksPath, w.eventsPath, w.Write(ctx)
+}
+
+// SessionEvents returns slice of event fields from gzipped events file.
+// The file at eventsPath will be removed.
+func SessionEvents(eventsPath string) ([]EventFields, error) {
```

Comment:
Actually thinking about this some more - what would stop you from implementing these on `PlaybackWriter` since it already has access to both filepaths. Also - they don't need to take up the (actually caller's) responsibility of removing the files - the files could be written to a temporary directory and then the whole directory could be cleaned up by the caller?

Reply:
I could implement on `PlaybackWriter`, however the caller (`PlayFromFile`), doesn't create an instance of it, an instance of playback writer is created within `WriteForPlayback()`.I suppose instead of returning the paths I could return that instance of `PlaybackWriter`? and then the caller can remove the files and call `SessionEvents` and `SessionChunks` without passing in the path.

Reply:
Yes - exactly what I'm suggesting.